src/components/TopicsCard.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function TopicsCard({topic,handleArticleByTopic}) {
+  return (
+    <div className='flex p-4 m-2 border border-solid border-green-500 flex-col items-center'>
+        <p>Slug: <span className='font-semibold'>{topic.slug}</span></p>
```

Comment:
It might be more user-friendly to have the buttons marked as each topic, eg 'football', 'coding' and 'topics' rather than including the slug and description with a button underneath. You could have one dropdown including all the topics for the user to select 💡